### PR TITLE
Add LongStreetNodes to registry.

### PR DESCRIPTION
**Pull request by Lazydayz137**
Adding LongStreetNodes to registry.

### CODE REVIEW

**Comment by Lazydayz137** [comment]
Something wrong that it didn't get merged with the others?  Apologies if so.

File: LongStreetNodes/chains.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{
+  "$schema": "../chains.schema.json",
+  "name": "Long Street Nodes.",
+  "chains": [
+    {
+      "name": "bitcanna",
+      "address": "bcnavaloper1gwcxmqwzjvgd9lwd3tckrvy8nj5rmxp2k0z5md",
+      "restake": {
+        "address": "bcna1rzmj90ucld9pkzp05flkjtjvl783982alhmz28",
+        "run_time": ["every 5 minutes"],
```
Comment (tombeynon) [inline LongStreetNodes/chains.json:10]:
```suggestion
        "run_time": "every 5 minutes",
```

File: LongStreetNodes/chains.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+        "minimum_reward": 100000000000000000
+      }
+    },
+    {
+      "name": "acrechain",
+      "address": "acrevaloper1s5u7zzcrklkfe4wdmxt080e5u58sknevrpkmuw",
+      "restake": {
+        "address": "acre13qg7urur85u3g0l0097pwl9ap3vkgzykynlnk3",
+        "run_time": ["every 5 minutes"],
```
Comment (tombeynon) [inline LongStreetNodes/chains.json:19]:
```suggestion
        "run_time": "every 5 minutes",
```

**Comment by Lazydayz137** [comment]
You are a scholar and a gentleman.  Completely missed the formatting error.  Apologies and thank you!

**Pull request merged**